modules/nw-ipfailover-configuration.adoc

```diff
@@ -0,0 +1,180 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ipfailover.adoc
+
+[id="nw-ipfailover-configuration_{context}"]
+= Configuring IP failover
+
+As a cluster administrator, you can configure IP failover on an entire cluster, or on a subset of nodes, as defined by the label selector. You can also configure multiple IP failover deployment configurations in your cluster, where each one is independent of the others.
+
+The IP failover deployment configuration ensures that a failover pod runs on each of the nodes matching the constraints or the label used.
+
+This pod runs Keepalived, which can monitor an endpoint and use Virtual Router Redundancy Protocol (VRRP) to fail over the virtual IP (VIP) from one node to another if the first node cannot reach the service or endpoint.
+
+For production use, set a `selector` that selects at least two nodes, and set `replicas` equal to the number of selected nodes.
+
+.Prerequisites
+
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* You created a pull secret.
+
+.Procedure
+
+//. Create an {product-title} pull secret
+//+
+. Create an IP failover service account:
++
+[source,terminal]
+----
+$ oc create sa ipfailover
+----
++
+. Update security context constraints (SCC) for `hostNetwork`:
++
+[source,terminal]
+----
+$ oc adm policy add-scc-to-user privileged -z ipfailover
+$ oc adm policy add-scc-to-user hostnetwork -z ipfailover
+----
++
+. Create deployment and service YAML files to configure IP failover:
```

Comment:
If the service is removed, this needs to change to "Create deployment YAML file to configure IP failover"